spring-integration-hazelcast/src/main/java/org/springframework/integration/hazelcast/common/CacheEventType.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.hazelcast.common;
+
+/**
+ * Enumeration of Cache Event Types of Hazelcast Inbound Channel Adapter
```

Comment:
No reason to mention `Hazelcast Inbound Channel Adapter` from the description JavaDoc.
You can use `@see AbstractHazelcastMessageProducer`  after `@since`